Resources/frontend/js/jquery.adyen-confirm-order.js

```diff
@@ -52,6 +52,8 @@
             var me = this;
             if (!$.isEmptyObject(me.opts.adyenSetSession)) {
                 me.sessionStorage.setItem(me.paymentMethodSession, JSON.stringify(me.opts.adyenSetSession));
+            } else {
+                me.sessionStorage.removeItem(me.paymentMethodSession);
```

Comment:
This is necessary to clear the sessionStorage data when it is not sent from the backend, otherwise this widget might still attempt to pay with Adyen (in `onPlaceOrder`) and fail.